### ISSUE TITLE
JetNews doesn't work with a screen reader

### ISSUE THREAD

**Issue by shailen** [android/compose-samples#5]
Very nice app, but it needs to support blind and low-vision users using a screenreader like Talkback. Right now, the app is completely opaque to such users. 

To repro, turn on Talkback from Settings > Accessibility and launch JetNews. The title of the app is announced; no other app content is accessible.
